utilities/storage.py

```
@@ -606,12 +607,23 @@ def get_test_artifact_server_url(schema="https"):
+            sleep=TIMEOUT_5SEC,
+            func=lambda: requests.get(
+                artifactory_connection_url, headers=utilities.infra.get_artifactory_header(), verify=False
+            ),
+        ):
+            if sample.status_code == requests.codes.ok:
+                return py_config["servers"][f"{schema}_server"]
+    except TimeoutExpiredError:
+        LOGGER.error(
+            f"Unable to connect to test image server: {artifactory_connection_url} "
+            f"{schema.upper()}, with error code: {sample.status_code}, error: {sample.text}"
+        )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)